sceptre/cli/list.py

```diff
@@ -127,3 +126,42 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.command(name="stacks")
+@click.option(
+    "--all", is_flag=True, help="List all stacks in stack groups."
+)
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path, all):
+    """
+    List stack names and command paths for stacks in a
+    stack group.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    :param all: List all stacks in stack groups.
+    :type all: bool
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        output_format=ctx.obj.get("output_format"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies"),
+        full_scan=True
```

Comment:
I thought you said `full_scan` wasn't doing anything for you?

Reply:
I don't think you want to set full_scan to True here otherwise sceptre will always generate a list for all stack groups no matter what path you pass

Reply:
Ah, ok... it seemed to work fine but I'll change it back.

Reply:
you didn't change it back?

Reply:
@zaro0508  I  did now, I must have somehow missed this.